Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -2011,9 +2011,12 @@ static bool isContextMenuEvent(const WebMouseEvent& event)
 
 void UnifiedPDFPlugin::repaintAnnotationsForFormField(NSString *fieldName)
 {
+#if HAVE(PDFDOCUMENT_ANNOTATIONS_FOR_FIELD_NAME)
     RetainPtr annotations = [m_pdfDocument annotationsForFieldName:fieldName];
     for (PDFAnnotation *annotation in annotations.get())
         setNeedsRepaintForAnnotation(annotation, repaintRequirementsForAnnotation(annotation));
+#endif
+    UNUSED_PARAM(fieldName);
```

Comment:
Should this be in an `else` in case we compile the code above?

Reply:
Sure, it's benign even if the above code gets compiled though.

Reply:
Fixed in [2cac936](https://github.com/WebKit/WebKit/commit/2cac9367870b2dd92ff126875a2c842aea56e7b4)